CONTRIBUTING.md

```diff
@@ -1,3 +1,70 @@
-Welcome to the Kong/charts repo! We're currently in the process of writing a more detailed CONTRIBUTING guide.
+# Contributing to Kong Helm charts
 
-In lieu of a complete guide, please base your pull requests off the `next` branch and do not increment your chart's version number as part of your changes. The repo maintainers will bundle changes together periodically and bump the version when merging changes from `next` into `master`.
+Feel free to contribute fixes or minor features, we love to receive pull
+requests! If you are planning to develop a larger feature, please come talk to
+us first. We can be found in the [#kong
+channel](https://kubernetes.slack.com/archives/CDCA87FRD) on the Kubernetes
```

Comment:
Please instead mention issues on this repo itself. We want to cut down on slack conversations around features. Github issues work better for feature requests and conversations requiring slow conversations, and we get a better archeology of our work.